aiida_vasp/utils/aiida_utils.py

```diff
@@ -87,6 +89,51 @@ def builder_interface(calc_cls):
     return False
 
 
+def copy_parameter(old_parameter):
+    """Assemble a new ParameterData."""
+    return get_data_node('parameter', dict=old_parameter.get_dict())
+
+
+def displaced_structure(structure, displacement, entry):
+    disp_structure = structure.clone()
+    displace_position(disp_structure, displacement, entry)
+    return disp_structure
+
+
+def compressed_structure(structure, volume_change):
+    comp_structure = structure.clone()
+    compress_cell(comp_structure, volume_change)
+    return comp_structure
+
+
+def displace_position(structure, displacement, entry):
+    """Displace a position in the StructureData."""
+    sites = structure.sites
+    positions = []
+    for site in sites:
+        positions.append(site.position)
+    position = positions[entry]
+    position = position + displacement
+    structure.reset_sites_positions(positions)
+
+
+def compress_cell(structure, volume_change):
+    """Apply compression or tensile forces to the unit cell."""
+    cell = structure.cell
+    new_cell = np.array(cell) * volume_change
+    structure.reset_cell(new_cell.tolist())
+
+
+def not_ubuntu():
```

Comment:
I think there is a conflict between the naming of the function and the return values. i.e. 'not_ubuntu' returns True if it is Ubuntu and False if it is not Ubuntu. I suggest renaming it to 'is_ubuntu' or reversing the return values.

Reply:
Okay my first comment was wrong because I got confused by the double inversion. Actually the return value 'False' here means `(not linux) or (ubuntu)`, so `not not_ubuntu` does not mean we are using ubuntu.

Reply:
It may become obsolete after #223 since that includes also a workaround for the 'direct_transport' issue.

Reply:
Yes, all this is in fact not so valid anymore as we have fixes to make it run on other Linux systems than Ubuntu now. And you have fixes for the mac os, so then I think we do not need any of these. I will remove them.